### PR TITLE
Update James SHA-1 (postgresql)

### PR DESCRIPTION
**Pull request by vttranlina**
resolve https://github.com/linagora/tmail-backend/issues/981

### CODE REVIEW

**Comment by Arsnael** [comment]
Really that simple?

**Comment by vttranlina** [comment]
> Really that simple?

Yeah, 
I resolved some conflicts and force push to `postgresql` branch
This PR to see the CI result

**Comment by vttranlina** [comment]
last commit for Upgrade Jooq version

File: tmail-backend/apps/postgres/src/main/java/com/linagora/tmail/james/app/PostgresTmailServer.java [hunk 1]
```diff
@@ -148,6 +149,7 @@ public static GuiceJamesServer createServer(PostgresTmailConfiguration configura
 
     public static final Module JMAP_LINAGORA = Modules.override(
         JMAP,
+            new TMailJMAPModule(),
```
Comment (Arsnael) [inline tmail-backend/apps/postgres/src/main/java/com/linagora/tmail/james/app/PostgresTmailServer.java:152]: indent

**Comment by Arsnael** [comment]
Green! Fix the indent, and squash me everything into a beautiful git commit history, and we can merge :)

**Pull request merged**